### ISSUE TITLE
What does the ones means?

### ISSUE THREAD

**Issue by jerrywyn** [msxie92/MangaInpainting#6]
At the networks.py line 218,         
```
ones = torch.ones_like(mask)
# conv branch
xnow = torch.cat([xin, ones, mask], dim=1)
```
Does mask contatins all info of ones? Why add this in the network input?
